### DISCUSSION TITLE
絵文字の背景が白塗りなのでダークモードで違和感がある

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#466] [要望・アイデア・不具合]
絵文字の背景が白塗りなので、ダークモードで見たときに違和感があります。

通常:

<img width="552" alt="スクリーンショット 2023-04-23 21 31 59" src="https://user-images.githubusercontent.com/20086673/233840008-234602a3-6ace-4330-9ce9-40803de73960.png">

ダークモードを適用:

<img width="552" alt="スクリーンショット 2023-04-23 21 29 21" src="https://user-images.githubusercontent.com/20086673/233839886-3dfca03f-1b16-4132-85a1-4ab5cef28ffb.png">

https://qiita.com/Qiita/items/c686397e4a0f4f11683d

絵文字の背景は透過するのがよさそうです。

**Comment by tanaka-asami415** [comment]
@munierujp 
ご意見ありがとうございます🙏
使用しているライブラリの都合上、すぐに対応することは難しそうですが検討します！

**Comment by wataru86** [comment]
@munierujp

お待たせして申し訳ありません。

下記のリリース時に絵文字の背景色も修正しました。
ご報告が漏れており、大変失礼いたしました🙇
https://qiita.com/release-notes/2023-06-20

**Reply by munierujp** [reply]
いえいえ、ご対応いただきありがとうございます😊
